### PR TITLE
Form Error + Loading + Success States

### LINKED ISSUE #3262: Add positive feedback to text fields and long forms
### Expected Behavior
Per circle feedback let's roll this out with just Jost Utilitarian responses. 

In the future I can see us having more whimsical/conversational states for markety/experiential type experiences 

image
image

### Current Behavior

No feedback as of now.

### PR DESCRIPTION
**Pull request by aaronmgdr**
### Description

1. Completes the unification of form errors states

2. adds a unified form success state

3. adds  an isLoading state with a spinner on all form submit buttons

### Tested

existing tests, plus some new tests, minimal changes to tests.
Screen Shot 2020-04-02 at 5 37 13 PM
Screen Shot 2020-04-02 at 5 37 21 PM
Screen Shot 2020-04-02 at 5 37 30 PM
Screen Shot 2020-04-02 at 5 39 27 PM

Screen Shot 2020-04-02 at 5 36 12 PM
Screen Shot 2020-04-02 at 5 36 58 PM

### Other changes

* removes unused old submit function and changes altSubmit on Form to be onSubmit

* changes yarn test to not always run coverage. 

* Moves `Checkbox` and `TextInput` to own files, 

### Related issues

- Fixes https://github.com/celo-org/celo-monorepo/issues/3262
- fixes https://github.com/celo-org/celo-labs/issues/410

### CODE REVIEW

**Comment by aaronmgdr** [comment]
Looks big but keep in mind that over half of that is snapshot changes due to how Error and Success States are now handled

File: packages/web/src/forms/CheckboxWithLabel.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    <View style={checkBoxStyles.border}>
+      <Text
+        style={[
+          checkBoxStyles.checkMark,
+          checked ? checkBoxStyles.checkMarkChecked : checkBoxStyles.hidden,
+        ]}
+      >
+        ✓
+      </Text>
+      {createElement('input', {
```
Comment (jmrossy) [inline packages/web/src/forms/CheckboxWithLabel.tsx:44]: Is this considered unstable? https://github.com/necolas/react-native-web/issues/1385

**Review by jmrossy — COMMENTED** [review]
The createElement call looks off to me but otherwise all looks good.
Also looks like general tests are failing atm.

File: packages/web/src/forms/CheckboxWithLabel.tsx (hunk 1 above)
Comment (aaronmgdr) [reply packages/web/src/forms/CheckboxWithLabel.tsx:44]: oh no, 

we use this in other places too.

Comment (aaronmgdr) [reply packages/web/src/forms/CheckboxWithLabel.tsx:44]: given that there are other places we will have to fixes this when upgrading RNW and that Checkbox used this before and was just moved, id like to ask that we merge in for now. 

But good to know.

**Review by jmrossy — APPROVED** [review]

File: packages/web/src/forms/CheckboxWithLabel.tsx (hunk 1 above)
Comment (jmrossy) [reply packages/web/src/forms/CheckboxWithLabel.tsx:44]: Ok 👍

**Pull request merged**